plugins/org.eclipse.fordiac.ide.fb.interpreter/src/org/eclipse/fordiac/ide/fb/interpreter/testappgen/TestGenBlockNames.java

```diff
@@ -1,10 +1,25 @@
+/*******************************************************************************
+ * Copyright (c) 2023, 2025 Johannes Kepler University Linz
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Melanie Winter
+ *     - initial API and implementation and/or initial documentation
+ *   Bianca Wiesmayr - updated identifiers
+ *******************************************************************************/
+
 package org.eclipse.fordiac.ide.fb.interpreter.testappgen;
 
 public class TestGenBlockNames {
 
-	public static final String TIMEOUT_ADAPTER_NAME = "ATimeOutS"; //$NON-NLS-1$
+	public static final String TIMEOUT_ADAPTER_NAME = "iec61499::events::atimeout"; //$NON-NLS-1$
```

Comment:
Although IEC 61499 is case insensitive it may be nicer to use the same case style as in the type: `ATimeOut`.